oap-ws-sso-api/src/main/java/oap/ws/sso/Authentication.java

```diff
@@ -38,10 +38,12 @@
     public final String id;
     public final User user;
     public DateTime created;
+    public boolean tfaEnabled;
 
-    public Authentication( String id, User user ) {
+    public Authentication( String id, User user, boolean tfaEnabled ) {
```

Comment:
For the BC please add an additional constructor with 2 arguments and default tfaEnabled param